libraries/AP_RangeFinder/AP_RangeFinder_uLanding.h

```diff
@@ -24,11 +24,17 @@ class AP_RangeFinder_uLanding : public AP_RangeFinder_Backend
     }
 
 private:
+    // detect uLanding Firmware Version
+    bool detect_uLanding_version(void);
+
     // get a reading
     bool get_reading(uint16_t &reading_cm);
 
     AP_HAL::UARTDriver *uart = nullptr;
     uint32_t last_reading_ms = 0;
-    uint8_t linebuf[10];
+    uint8_t linebuf[6];
     uint8_t linebuf_len = 0;
+    bool _uLanding_version_check = false;
```

Comment:
The initialization isn't needed, the whole object is zeroed on construction.

Also, can we change the variable name? To me, the variable indicates that when it is true it still needs to check the version, but it actually is working the other way around; maybe `version_known`?

Reply:
Good call, I thought that name felt a little odd, but end-of-day me couldn't think of something better.